packages/pangea-sdk/pangea/services/redact.py

```diff
@@ -107,7 +107,7 @@ class Redact(ServiceBase):
     """Redact service client.
 
     Provides the methods to interact with the Pangea Redact Service:
-        [https://docs.dev.pangea.cloud/docs/api/redact](https://docs.dev.pangea.cloud/docs/api/redact)
+        [https://pangea.cloud/docs/api/redact](https://pangea.cloud/docs/api/redact)
```

Comment:
need to keep the service name in the domain

Reply:
according to @r3dcrosse it's ok